### PR TITLE
chore: Employee Doctype Enhancement

### PR DESCRIPTION
**Pull request by mymi14s**
## Feature description
Minor changes to Employee Doctype form presentation.

image

### CODE REVIEW

File: one_fm/public/js/doctype_js/employee.js [hunk 1]
```diff
@@ -1,11 +1,54 @@
...
+			}
+			// end modify bank account link
+			// add bank account button
+			frm.add_custom_button('Bank Account', ()=>{
+				frappe.new_doc('Bank Account', {
+					party_type:frm.doc.doctype,
+					party:frm.doc.name,
+					account_name:frm.doc.employee_name,
+				})
+			}, 'Create');
```
Comment (pjamsheer) [inline one_fm/public/js/doctype_js/employee.js]: @mymi14s @taqi-onefm Creating bank account is not permitted for the employee. It is having a workflow, first the employee needs to request for new bank account. Or the bank account will be created form the Onboarding process.

**Review by pjamsheer — COMMENTED** [review]
@mymi14s @taqi-onefm PAM File number is linked with Contract File and PAM Authorized Signatory List. So we need to add that linking.
Screenshot 2022-04-05 at 1 02 43 PM

File: one_fm/public/js/doctype_js/employee.js (hunk 1 above)
Comment (mymi14s) [reply one_fm/public/js/doctype_js/employee.js]: Ok

**Comment by mymi14s** [comment]
@pjamsheer issue resolved.

**Pull request merged**